RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,183 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_Constants_h
+#define RecoTracker_LSTCore_interface_alpaka_Constants_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+#ifdef CACHE_ALLOC
+#include "HeterogeneousCore/AlpakaInterface/interface/CachedBufAlloc.h"
+#endif
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+#include <cuda_fp16.h>
+#endif
+
+namespace SDL {
```

Comment:
Namespaces should be in lower case. It is also unclear (at least for a casual reader) what "SDL" stands for. I would have expected e.g. `namespace lst`.

Reply:
For me SDL is https://www.libsdl.org/ .

Reply:
The CMS naming convention uses lower case for namespaces.

Reply:
super-doublet linking (SDL) is the original acronym/initialism.
LST was settled following "external"/CMS performance review inputs some time after the initial code was written.